apps/web/frontend/components/weekTable/weekTableTaskRow.tsx

```diff
@@ -13,17 +13,41 @@ export const WeekTableTaskRow = ({ interval, task }: WeekTableTaskRowProps) => {
     .map((workHour) => workHour.duration)
     .reduce((previous, current) => previous + current, 0)
 
+  const [{ data, error, fetching: projectsLoading }] = useMyProjectsQuery({
+    variables: { from: format(new Date(), 'yyyy-MM-dd'), filter: ProjectFilter.All },
+  })
+
+  const currentProjekt = data?.projects.find((project) => project.id === task.project.id)
+  const startDay = new Date(currentProjekt?.startDate ?? '')
+  const endDate = new Date(currentProjekt?.endDate ?? '')
+
   return (
     <TableRow>
       <TableCell>{task.title}</TableCell>
-      {eachDayOfInterval(interval).map((day) => {
-        const durations = task.workHours
-          .filter((workHour) => isSameDay(parseISO(workHour.date), day))
-          .map((workHour) => workHour.duration)
-        const duration = durations.reduce((previous, current) => previous + current, 0)
+      {error && <span>{error.message}</span>}
+      {projectsLoading && <Spinner />}
```

Comment:
I think it is forbidden to put any element other than `TableCell` in a `TableRow`.